docs/getting-started-tutorial/1-view-first-development.adoc

```diff
@@ -0,0 +1,71 @@
+:idprefix:
+:idseparator: -
+:toc: right
+:toclevels: 2
+
+# View-first Development
+
+If you're developing a user-facing web site or application, one of Lift's
+greatest improvements over existing systems is view-first development.
+View-first development thoroughly separates the process of creating the user
+interface from the process of putting data from the system into it, in a way
+that lets you stay focused on users when you're creating the user interface and
+worry about the the interface between your backend and the HTML only when
+you're working on the backend.
+
+The flip side of view-first development is that it takes some getting used to
+if one is accustomed to the typical web MVC framework. The first stop when
+figuring out what's going on in a typical web MVC setup is the controller. In
+Lift, your first stop is your HTML file. Everything starts in the HTML, and in
+what it is that you want to present to the user. You don't just think about
```

Comment:
`Everything starts in the HTML, and in what it is that you want to present to the user.`

This sentence took me a few reads to comprehend correctly. It might be a good candidate for some tweaking.

Reply:
Rephrased, great point.